README.md

```diff
@@ -173,46 +192,98 @@ To build the modern eBPF probe, you need:
 
 ```bash
-cmake -DUSE_BUNDLED_DEPS=ON -DBUILD_LIBSCAP_MODERN_BPF=ON -DMODERN_BPF_SKEL_DIR="/tmp/skel-dir" -DBUILD_LIBSCAP_GVISOR=OFF .. 
+cmake -DUSE_BUNDLED_DEPS=ON -DBUILD_LIBSCAP_MODERN_BPF=ON -DMODERN_BPF_SKEL_DIR="/tmp/skel-dir" .. 
 ```
 
 ### gVisor support
 
 Libscap contains additional library functions to allow integration with system call events coming from [gVisor](https://gvisor.dev).
-Compilation of this functionality can be disabled with `-DBUILD_LIBSCAP_GVISOR=Off`.
+Compilation of this functionality can be disabled with `-DBUILD_LIBSCAP_GVISOR=OFF`.
+
+</details>
+
+</br>
+
+## Testing
+
+<details>
+	<summary>Expand Testing Instructions</summary>
+
+This repository includes convenient test example binaries for both `scap` and `sinsp`:
+
+* `scap-open` -> build a small example binary for `libscap` to test the drivers (dependent on `scap`), checkout the program's [documentation](./userspace/libscap/examples/01-open/README.md)
+* `sinsp-example` -> build a small example binary for `libsinsp` to test the drivers and/or `libsinsp` functionality (dependent on `scap` and `sinsp`), checkout the program's [documentation](./userspace/libsinsp/examples/README.md)
+
+When developing new features, you would run either one depending on what you're working on, in order to test and validate your changes.
+
+> __NOTE:__ When you're working on driver development, it can be quite useful to make use of the kernel's built-in `printk` functionality. We also provide build flags to enable `BPF_DEBUG`. However, for the traditional bpf driver, you'll need to uncomment a line in the [bpf Makefile](driver/bpf/Makefile) first. Any logs generated by `bpf_printk()` will be written to `/sys/kernel/debug/tracing/trace_pipe`. Just make sure you have the right permissions set up for this.
+
+Here's an example of a `cmake` command that will enable everything you need for all tests and components:
+
+```bash
+cmake -DUSE_BUNDLED_DEPS=ON -DBUILD_LIBSCAP_MODERN_BPF=ON -DBUILD_MODERN_BPF_TEST=ON -DMODERN_BPF_DEBUG_MODE=ON -DBUILD_LIBSCAP_GVISOR=ON -DBUILD_BPF=ON -DBUILD_DRIVER=ON -DBPF_DEBUG=ON -DCREATE_TEST_TARGETS=ON -DENABLE_LIBSCAP_TESTS=ON -DENABLE_DRIVERS_TESTS=ON -DENABLE_VM_TESTS=ON ../;
+```
+
+```bash
+rm -f driver/bpf/probe.o; make bpf;
+rm -f driver/src/scap.ko; make driver;
+rm -f libscap/examples/01-open/scap-open; make -j 16 scap-open;
+rm -f libsinsp/examples/sinsp-example; make -j 16 sinsp-example;
+```
+
+These are the conventional unit tests that our CI system enforces:
+
+```bash
+make -j 16 unit-test-libsinsp; make run-unit-test-libsinsp;
+make -j 16 run-unit-test-libscap; sudo ./libscap/test/unit-test-libscap;
```

Comment:
i think we don't this anymore, all these tests have been ported to `make linscap_test`